src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/MongoDbSessionHandlerTest.php

```diff
@@ -54,163 +59,141 @@ protected function setUp(): void
             'collection' => 'session-test',
         ];
 
-        $this->storage = new MongoDbSessionHandler($this->mongo, $this->options);
+        $this->storage = new MongoDbSessionHandler($this->manager, $this->options);
     }
 
-    public function testConstructorShouldThrowExceptionForMissingOptions()
+    public function testCreateFromClient(): void
     {
-        $this->expectException(\InvalidArgumentException::class);
-        new MongoDbSessionHandler($this->mongo, []);
+        if (!class_exists(Client::class)) {
+            $this->markTestSkipped('The "mongodb/mongodb" library is required.');
```

Comment:
This test still requires the `mongodb/mongodb` library. It will not run in the CI. In order to mock the Client class, what do you think of creating stubs?

Reply:
Added minimal stubs.